doc/source/ray-more-libs/dask-on-ray.rst

```diff
@@ -224,8 +224,7 @@ The following Ray-specific callbacks are provided:
    6. :code:`ray_finish(result)`: Run after all Ray tasks have finished
       executing and the final result has been returned.
 
-See the docstring for
-:meth:`RayDaskCallback.__init__() <ray.util.dask.callbacks.RayDaskCallback>.__init__`
+See the docstring for :meth:`~ray.util.dask.RayDaskCallback.__init__`
```

Comment:
qq, what the `~` means?

Reply:
It tells Sphinx to not include the `ray.util.dask.RayDaskCallback` part when the link is rendered. So the link will appear as just `__init__()` in the docs.